packages/components/src/font-size-picker/index.js

```diff
@@ -15,23 +15,45 @@ import { __ } from '@wordpress/i18n';
 import Button from '../button';
 import ButtonGroup from '../button-group';
 import RangeControl from '../range-control';
+import Tooltip from '../tooltip';
+
+/**
+ * If short and long font size names are available, render using the short name and add the tooltip using the long name.
+ * If only short or long names but not their counterparts are provided, render them using only what is provided.
+ *
+ * @param {string}   label    Font size name, can be long, Small, or short, S.
+ * @param {string}   value    Current font size.
+ * @param {string}   size     Font size that this button sets.
+ * @param {function} onChange Method that sets the font to the specified size.
+ *
+ * @return {Element} Element with name with tooltip.
+ */
+const fontSize = ( label, value, size, onChange ) => {
```

Comment:
I think this function was just extracted to reuse the button. In that case, can we just assign inside the map function a `const fontSizeButton = <Button ...` and then wrap it in a tooltip if a name and short name exists?
Something like:
```
					{
						map( fontSizes, ( { name, size, shortName } ) => {
							const fontSizeButton = <Button ...
							if ( shortName && name ) {
								return ( <Tooltip...> { fontSizeButton } </Tooltip> );
							}
							rerturn fontSizeButton;
						}
					}
```

Reply:
Keeping the same logic is also a possibility but in this case, I think we should rename the function to specify its intent something like renderFontSizePickerButton or similar.